src/ant/src/AntennaChecker.cc

```diff
@@ -193,21 +185,6 @@ void AntennaChecker::initAntennaRules()
       const dbTechLayerAntennaRule* antenna_rule
           = tech_layer->getDefaultAntennaRule();
 
-      odb::dbStringProperty* layer_prop = odb::dbStringProperty::find(
-          tech_layer, "LEF57_ANTENNAGATEPLUSDIFF");
```

Comment:
It turns out there we do not need to look for the property in the LEF file ourselves because it is accesible  through the ```getGatePlusDiffFactor()```  funciton from the LEF parser. This function is called in line 207.